lib/options.json

```diff
@@ -92,21 +92,16 @@
     "description": "Determines whether console output should include colors.",
     "default": true
   },
-  "level": {
+  "loglevel": {
     "alias": "l",
-    "description": "The level of logging to output. Options: silly, debug, verbose, info, warn, error.",
-    "default": "info"
+    "description": "The level of logging to output. Options: 'debug', 'warning', 'error', 'silent'.",
+    "default": "warning"
   },
   "timestamp": {
     "alias": "t",
     "description": "Determines whether console output should include timestamps.",
     "default": false
   },
-  "silent": {
```

Comment:
Wondering, would be possible / make sense for us to detect this argument still and map it to log level and provide a deprecation warning to the user? Should make it simpler for users to upgrade and understand why some arguments no longer work.

Reply:
I'm intentionally doing evil here by skipping the deprecation period for things to allow myself a major series of breaking changes to tidy up Dredd's codebase.

The major release version should be a warning for people to read the changelog and there are always clear steps they need to take to make their setup work.

So yes, I could deprecate this gradually, but I'm afraid I'd end up managing deprecation warnings in the codebase instead of going forward with cleaning up the project from technical debts the fastest way I can. What do you think of this? Is it way too evil?

Reply:
As a user it would be nice, it doesn’t have to be for long though. 3-6 month period. It’s up to you though not blocking on this.

Reply:
Addressed in 01b042e